cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1342,6 +1410,16 @@ instance (Show addr, Show versionNumber, Show agreedOptions, ToObject addr)
           [ "kind" .= String "ConnectionManagerCounters"
           , "state" .= String (pack . show $ cmCounters)
           ]
+      TrState cmState ->
+        mkObject
+          [ "kind"  .= String "State"
```

Comment:
Can you change `State` to something more grep:able.

Reply:
Sure, `connectionManagerState` should be easier to grep.

Reply:
Done